packages/react/src/components/UIShell/HeaderMenu.js

```diff
@@ -53,8 +47,14 @@ class HeaderMenu extends React.Component {
     tabIndex: PropTypes.number,
   };
 
+  static contextType = PrefixContext;
+
+  static defaultRenderMenuContent = () => (
+    <ChevronDown16 className={`${this.context}--header__menu-arrow`} />
```

Comment:
When it's a static method, I don't think `this.context` will be defined (because it's static).

Down below, what you could do instead is `MenuContent ? <MenuContent /> ? <ChevronDown16 className={`${this.context}--header__menu-arrow`} />` and I think that would work